spec/classes/php_spec.rb

```diff
@@ -292,6 +292,25 @@
       end
 
       if facts[:osfamily] == 'RedHat' || facts[:osfamily] == 'CentOS'
+        describe 'when called with valid settings parameter types' do
```

Comment:
Why is this new test in this `if` block for only RedHat and CentOS? Seems like it should apply to any OS.

Reply:
I said it in another comment -- but it doesn't appear to work outside that block and to be frank I don't have a lot of time to debug why this week.  =)  I copied what was already being done and adapted it.  I suspect it has to do with the /etc/php.ini reference and that being different on Ubuntu and such but ???  

As far as my email address -- I went ahead and associated my work email with github.com as well so no worries there.  Thanks!

Reply:
Yeah, looks like the spec file would have to introduce a variable to mimic the `$config_root_inifile` puppet variable, since that path is used in the resource names (similar to the case statements used for the package names). I had the idea of using a regexp for the resource title, but looks like you can't do that, but you can test the catalog directly: https://github.com/rodjek/rspec-puppet/issues/431

As long as this module supports RHEL/CentOS, which seems very likely to always be the case, having these tests only running under RHEL/CentOS seems good enough for now.